content/actions/managing-workflow-runs/downloading-workflow-artifacts.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Downloading workflow artifacts
 intro: You can download archived artifacts before they automatically expire.
+permissions: Authorized accounts with read access to a repository can download artifacts.
```

Comment:
Thanks so much for making that change @TheNH813! I'm going to tweak it just slightly to really reinforce the fact that you need to be signed in as you so helpfully pointed out. 💖 

```suggestion
permissions: People who are signed into {% data variables.product.product_name %} and have read access to a repository can download workflow artifacts.
```